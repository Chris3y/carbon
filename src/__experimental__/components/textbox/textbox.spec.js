import React from 'react';
import { shallow } from 'enzyme';
import Textbox from '.';

describe('Textbox', () => {
  it('renders with InputPresentation and Input and all props passed to Input', () => {
    const wrapper = shallow(
      <Textbox value='foobar' leftChildren='southpaw children'>
        normal children
      </Textbox>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
