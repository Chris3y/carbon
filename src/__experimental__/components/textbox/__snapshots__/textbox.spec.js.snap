// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textbox renders with InputPresentation and Input and all props passed to Input 1`] = `
<InputPresentation>
  southpaw children
  <Input
    value="foobar"
  />
  normal children
</InputPresentation>
`;
